packages/keystrokes/src/keystrokes.ts

```diff
@@ -368,27 +368,31 @@ export class Keystrokes<
     })
   }
 
-  private _handleKeyRelease(event: KeyEvent<OriginalEvent, KeyEventProps>) {
-    ;(async () => {
-      const key = event.key.toLowerCase()
+  private _releaseKey(event: KeyEvent<OriginalEvent, KeyEventProps>) {
```

Comment:
Moved some of this into a separate `_releaseKey` method, used internally when self-releasing keys as well as in `_handleKeyRelease`.  No logic was changed, the code was just re-organized.

Reply:
In my pr I decided to use the existing method in the fixed _tryReleaseSelfReleasingKeys method. Let me know if you have any thoughts as to why this wouldn't be a good idea.